.git-blame-ignore-revs

```diff
@@ -0,0 +1,17 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Scala Steward: Reformat with scalafmt 3.7.2
+3a80d550794d1278a636cd4d4a2ec0f33976f4a2
```

Comment:
I think this commit hash will change once this PR is merged because of the rebase. I suggest we add the `.git-blame-ignore-revs` file after this one is merged, and add multiple commit hashes (e.g. OSGI refactor, the previous scalafmt commit) in a separate PR.

Reply:
> I think this commit hash will change once this PR is merged because of the rebase. I suggest we add the `.git-blame-ignore-revs` file after this one is merged, and add multiple commit hashes (e.g. OSGI refactor, the previous scalafmt commit) in a separate PR.

That makes sense, although I'm disappointed it won't be as straightforward to merge future PRs from Scala Steward bumping scalafmt to new versions as I thought (unless we merge its PR so quickly so that a rebase won't be needed, or Scala Steward understands "@scala-steward rebase").  FYI, I don't have time to update the PR branch today, so I'll let you do that.

Reply:
Hopefully new scalafmt versions don't cause many, or any, changes. This one seems unique because I guess the old version missed the scripts?

Note that I think even if a rebase isn't needed, I *think* GitGub still changes something in the commit when we merge+rebase to cause the hash to change. I think maybe the committer author/date updated to when it's merged, which always updates the hash? So no matter how fast you merge, the hash will always change.